bakerydemo/templates/base/demo_banner.html

```diff
@@ -0,0 +1,7 @@
+{% load wagtailadmin_tags %}
+
+<link rel="stylesheet" href="{% versioned_static 'css/demo-banner.css' %}">
+
+<div id="demo-banner">
+    <h1>This is a demo site. Content created here will be reset at <a href="https://mytime.io/3am/UTC">3am UTC</a>.</h1>
```

Comment:
On the copy – my preference would be for a much simpler "will be reset every day". I don’t think the specific time matters enough to warrant the maintenance effort of listing it here.

If we want to keep the time – then let’s try to use a service that doesn’t use a `.io` domain if we can.

Reply:
I think including a cadence is important, but agree that as it's not a time defined in the repo, we should be more vague. I've updated it to "daily" (and thus removed the link)

Reply:
👍 sounds cool to me